src/modules/metrics/metrics.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { Controller, Get, Inject } from '@nestjs/common';
+import { Registry } from 'prom-client';
+import { Public } from '../authentication/api/public.decorator.js';
+import { ReporterService } from './reporter/reporter.service.js';
+import { RollenArt } from '../rolle/domain/rolle.enums.js';
+import { DBiamPersonenkontextRepo } from '../personenkontext/persistence/dbiam-personenkontext.repo.js';
+
+@ApiTags('Metrics')
+@Controller({ path: 'metrics' })
+export class MetricsController {
+    public constructor(
+        @Inject(Registry)
+        private readonly registry: Registry,
+        @Inject(ReporterService)
+        private readonly reporterService: ReporterService,
```

Comment:
No need for the `Inject`, ReporterService is an injectable

```suggestion
        private readonly reporterService: ReporterService,
```